README.md

```diff
@@ -151,6 +153,16 @@ UWP can only be built on a PC. `CMake` must be manually run to generate project
 1. In Apps\Playground, run `npm run windows`.
 > Note: if you experience build issues for Apps\Playground related to autolinking, try running `npx react-native autolink-windows` in the Apps\Playground folder. You can also run `npm run windows-verbose` to view logging.
 
+#### React Native
+
+Regardless of whether the app is developed for Android, iOS or Windows, it is necessary to specify the flex property for the SafeAreaView and EngineView components so that the engine can be created.
+
+```
+<SafeAreaView style={{flex: 1}}>
+    <EngineView style={{flex: 1}} />
+</SafeAreaView>
+```
+
```

Comment:
Would it make more sense for this block to be included somewhere in Modules/@babylonjs/react-native/README.md?

Reply:
Sure. I think it would be helpful to refer to the react-native readme from here though. People like me starting to set up the playground app for the first time could miss the additional docs.